Makefile

```
@@ -0,0 +1,166 @@
+	$(DOCKER_COMPOSE) restart
+
+install: docker-compose.override.yml build up deps perm                                                ## Create and start docker containers
+
+uninstall: stop                                                                                        ## Remove docker containers
+	$(DOCKER_COMPOSE) rm -vf
+
+reset: uninstall install                                                                               ## Remove and re-create docker containers
+
+clear-cache: perm
+	$(EXEC) $(CONSOLE) cache:clear --no-warmup
+	$(EXEC) $(CONSOLE) cache:warmup
+
+shell:                                                                                                 ## Run app container in interactive mode
+	$(EXEC) /bin/bash
+
+clear: perm                                                                                            ## Remove all the cache, the logs, the sessions and the built assets
+	$(EXEC) rm -rf var/cache/*
+	rm -rf var/log/*
+	rm -rf public/build
+	rm -f var/.php_cs.cache
+
+clean: clear                                                                                           ## Clear and remove dependencies
+	rm -rf vendor node_modules
+
+
+##
+## Database
+##---------------------------------------------------------------------------
+
+db-diff: vendor                                                                                        ## Generate a migration by comparing your current database to your mapping information
+	$(EXEC) $(CONSOLE) doctrine:migration:diff
+
+db-migrate: vendor                                                                                     ## Migrate database schema to the latest available version
+	$(EXEC) $(CONSOLE) doctrine:migration:migrate -n
+
+db-rollback: vendor                                                                                    ## Rollback the latest executed migration
+	$(EXEC) $(CONSOLE) doctrine:migration:migrate prev -n
+
+db-fixtures: vendor                                                                                    ## Apply doctrine fixtures
+	$(EXEC) $(CONSOLE) doctrine:fixtures:load -n
+
+db-validate: vendor                                                                                    ## Check the ORM mapping
+	$(EXEC) $(CONSOLE) doctrine:schema:validate
+
+
+##
+## Assets
+##---------------------------------------------------------------------------
+
+watch: node_modules                                                                                    ## Watch the assets and build their development version on change
+	$(EXEC) yarn watch
+
+assets: node_modules                                                                                   ## Build the development version of the assets
+	$(EXEC) yarn dev
+
+assets-build: node_modules                                                                             ## Build the production version of the assets
```

`assets` and `assets-build` both building assets
maybe use `assets-prod` instead of `assets-build`?
```suggestion
assets-prod: node_modules                                                                             ## Build the production version of the assets
```